src/systems/hydrodynamics/Hydrodynamics.cc

```diff
@@ -78,14 +79,121 @@ class gz::sim::systems::HydrodynamicsPrivateData
   /// \brief Previous state.
   public: Eigen::VectorXd prevState;
 
+  /// \brief Prefious derivative of the state
   public: Eigen::VectorXd prevStateDot;
 
-  /// Link entity
+  /// \brief Use current table if true
+  public: bool useCurrentTable {false};
+
+  /// \brief Current table xComponent
+  public: std::string axisComponents[3];
+
+  public: std::shared_ptr<gz::sim::v7::components::EnvironmentalData>
```

Comment:
@arjo129 nit: do we need to specify the version namespace? It should be inlined.